templates.js

```diff
@@ -4,30 +4,42 @@ var blockDict = [ // eslint-disable-line no-unused-vars
         "blockName": "IF",
         "code": "if (i == true) {\n\t// do something \n}",
         "buttonColor": "#ff3399", // fuschia
+        "kind": "monaco.languages.CompletionItemKind.Function",
```

Comment:
This seems to be the same across all blocks; can this be hard coded in the autocomplete provider instead?